pkg/flag/flag.go

```diff
@@ -28,23 +28,34 @@ const (
 	TriggerControllerName = "tekton-triggers-controller"
 	TriggerWebhookName    = "tekton-triggers-webhook"
 
-	AnnotationPreserveNS = "operator.tekton.dev/preserve-namespace"
+	AnnotationPreserveNS      = "operator.tekton.dev/preserve-namespace"
+	LabelTaskProviderType     = "operator.tekton.dev/provider-type"
+	TaskProviderTypeCommunity = "community"
```

Comment:
`RedHatProvider` may read better since it can be applied to various reosources.

Reply:
im bit confused here. Are you suggesting that he key should be `RedHatProvider`, `CommunityProvider`, `CertifiedProvider`